lib/image_processor/preprocessor/opencv-photo2scan.js

```diff
@@ -39,10 +39,16 @@ function handleMatrix(im, outfile, callback, config) {
   return callback(null, outfile);
 }
 
+function drawContour(im, contours){
```

Comment:
Maybe call it `verboseDrawContour`, and maybe add color option so it's more like:
```javascript
function verboseDrawContour(im, contours, color, thickness){
  color = color || [0, 255, 0]
  thickness = thickness || 5
  for(var c = 0; c < contours.size(); ++c) {
    im.drawContour(contours, c, color, thickness);
  }
}
```